vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/transformer/HeadlessCrawlerTableTransformer.java

```diff
@@ -66,7 +68,24 @@ private void addSeeds(URI mainApplicationPage, CrawlController controller)
                 .map(mainApplicationPagePath::concat)
                 .map(relativeUrl -> UriUtils.buildNewUrl(mainApplicationPage, relativeUrl))
                 .map(URI::toString)
-                .forEach(controller::addSeed);
+                .forEach(pageUrl -> addSeed(controller, pageUrl));
```

Comment:
I propose to move this logic into adapted crawl controller

Reply:
please elaborate

Reply:
Code in this class stays as is, all the handling happens in the CrawlControler

Reply:
once we have another place that uses CrawlControler - okay, but for now not sure why we need this

Reply:
This will keep the code clean and transparent, it will be testable and stuff :)

Reply:
subjective statement :)